SourceX/DiabloUI/diabloui.cpp

```diff
@@ -438,11 +348,142 @@ void LoadUiGFX() {
 	LoadArt("ui_art\\heros.pcx", &ArtHero, 4);
 }
 
+// clang-format off
+#define BLANKCOLOR { 0, 0xFF, 0, 0 }
+// clang-format on
+
+void LoadFallbackPalette()
+{
+	// clang-format off
+	PALETTEENTRY fallback_palette[256] = {

```

Comment:
Consider making it `static` and `const`, so that it's placed in the data section and not copied when the method is called:

```c++
static const PALETTEENTRY fallback_palette[256]
```

Reply:
Thanks for the helpful description.